localisation/000_wc_military_localization.csv

```diff
@@ -51,14 +51,14 @@ HEAVY_INFANTRY_OFFENSIVE_FIRE;Infantry Attack;;Infanterie 
 HEAVY_INFANTRY_DEFENSIVE_FIRE;Infantry Defense;;Infanterie � Verteidigung;;;;;;;;;;;x
 HEAVY_INFANTRY_MORALE;Infantry Morale;;Infanterie � Moral;;;;;;;;;;;x
 HEAVY_INFANTRY_MODIFIER;Infantry Modifier;;Infanterie � Disziplin;;;;;;;;;;;x
-CU_HEAVY_INFANTRY;Melee Infantry;;Nahkampfinfanterie;;;;;;;;;;;x
+CU_HEAVY_INFANTRY;Melee Troops;;Nahkampfinfanterie;;;;;;;;;;;x
```

Comment:
reeeee

Reply:
Why you're reeing? Unit groups called troops. That's the rule. Heavy troops, light troops, mounted troops.

Reply:
I was just reeing for kicks. I agree with this change and all others in this pull request